.github/workflows/pmm-portal-tests.yml

```diff
@@ -8,8 +8,6 @@ on:
   push:
```

Comment:
Okay, this pipeline will run only at midnight as it should pass, with default parameters on dev-latest. But removed all automatic triggers for upgrade pipeline since version needs to be provided there.